.env.default

```diff
@@ -18,11 +18,14 @@ IMAGE_REDIS=redis:5-alpine
 IMAGE_DRIVER=zenika/alpine-chrome
 CLEAR_FRONT_PACKAGES=no
 ADD_PHP_EXT=
+#ADD_PHP_EXT=php7-pdo_pgsql postgresql-client postgresql-contrib  gnu-libiconv wkhtmltopdf
+COMPOSER_HOME_CACHE=.cache/composer
```

Comment:
Needs to check if path relative or full

Reply:
Composer changed removed in favour of https://github.com/skilld-labs/skilld-docker-container/pull/278